source/game/system/KPadController.hh

```diff
@@ -14,6 +14,7 @@ enum class ControlSource {
     Gamecube = 3,
     Ghost = 4,
     AI = 5,
+    Host = 6, // New, represents an external program
```

Comment:
This is incredibly pedantic, but at some point this won't be new, so could we just say this is a Kinoko addition?

Reply:
Fixed.